packages/design-system/src/molecules/header/desktop.tsx

```diff
@@ -1,19 +1,17 @@
-import { StyleSheet, View } from 'react-native'
+import { View } from 'react-native'
 import { ButtonV2 } from '../../atoms'
 import { container, margin, padding } from '../../styles'
 import { isLast } from '../../utils'
 import { Web3Button } from '../web3Button'
 import { HeaderProps } from './types'
 
-const styles = StyleSheet.create({
-  mainContainer: {},
-})
-
 export const DesktopHeader = ({
   logo,
   options,
+  desktopSuffix,
```

Comment:
We can call it just `suffix`, because `Desktop` keyword is already used in the name of the component.

```suggestion
  suffix,
```